### ISSUE TITLE
There is no alt button in macos

### ISSUE THREAD

**Issue by qballer** [zellij-org/zellij#265]
MacOS alt button is mapped out to be the options key. While I see the awesome new UI which has this fancy new Alt option I can not actually use it. Is it possible to map the Alt button to options in case we load in Mac?

**Comment by bbil** [comment]
I ran into the same issue (back on a Mac for work), the issue is not this application, it is MacOS itself. What you need to do is tell `Terminal` or `iTerm` to use the option key in a non-mac way.

iterm -- set left option key as `Esc+` (not sure why `Meta` doesn't work):
![image](https://user-images.githubusercontent.com/2916172/115482286-52fc9100-a21c-11eb-9d8c-86b319a8af06.png)
 
Terminal -- use option as meta key:
![image](https://user-images.githubusercontent.com/2916172/115482384-9525d280-a21c-11eb-8d42-238b6e2fab42.png)

**Comment by bbil** [comment]
Doing the above means you now can't enter special characters using option/alt, like `alt+j` -> triangle. But, that's not something I do ever.

**Comment by believer** [comment]
If you are using [kitty](https://sw.kovidgoyal.net/kitty/) set `macos_option_as_alt yes` in your kitty settings

**Comment by steamfan** [comment]
The problem that you can not enter special characters anymore depends on the keyboard you are using: on a german keyboard `alt + n` gives you `~`. This might be often used to abbreviate the home directory.

**Comment by chrislaing** [comment]
One solution to this would be to allow multiple modifier keys (e.g. Alt + Ctrl or Alt + Shift).

**Comment by trst** [comment]
> If you use [kitty](https://sw.kovidgoyal.net/kitty/) set `macos_option_as_alt yes` in your kitty settings. Unfortunately, this breaks entering things like `@` and `~` (Swedish keyboard layout).

@believer To add on here, the options for Kitty's `macos_option_as_alt` config are `left`, `right` and `both`. This way if you need access to special characters, you could set only the right <kbd>Option</kbd> key to act as <kbd>Alt</kbd> for instance. More details on the [Kitty docs](https://sw.kovidgoyal.net/kitty/conf.html?highlight=alt#opt-kitty.macos_option_as_alt).

**Comment by qballer** [comment]
https://github.com/alacritty/alacritty/issues/62#issuecomment-347552058
for alacritty users

**Comment by felipesere** [comment]
@qballer are you running on Mac or Linux? Using Alt (on the left) + `[` or `]` does not switch between panes...

**Comment by poliorcetics** [comment]
See #1343

**Comment by bmikaili** [comment]
Neither the remappings by @qballer nor @felipesere work for me. Wondering if others are having the same issue?

**Comment by merikan** [comment]
@bmikaili which terminal are you using? Take a look at https://zellij.dev/documentation/faq.html#i-am-a-macos-user-how-can-i-use-the-alt-key

**Comment by smileart** [comment]
I have my left option configured as alt in kitty `macos_option_as_alt left` but still can't use `alt+[` for stacked panes as was explained here: https://zellij.dev/news/stacked-panes-swap-layouts Any solutions or progress on this one? 🤔

**Comment by tgross35** [comment]
For anyone using wezterm, you need the following to get skipping by word:

```lua
config.keys = {
  {
    key = 'LeftArrow',
    mods = 'ALT',
    action = wezterm.action.SendKey {
      key = 'b',
      mods = 'ALT',
    },
  },
  {
    key = 'RightArrow',
    mods = 'ALT',
    action = wezterm.action.SendKey {
      key = 'f',
      mods = 'ALT',
    },
  }
}
```

However, I still have an issue with zellij that isn't present outside of it. Option+Left correctly goes one word to the left, but Option+Right enters some sort of mode that jumps to the next character (Option+Right then `a` will go to the next character `a` after that position), or deletes (Option+Right followed by Right deletes to the end of the line). This happens even when zellij is locked. Does anyone know why that would be?

**Comment by kushvinth** [comment]
For those who use Ghostty, adding the given line to the Ghostty config works

```
macos-option-as-alt = true
```

**Comment by sbaro** [comment]
> For those who use Ghostty, adding the given line to the Ghostty config works
> 
> ```
> macos-option-as-alt = true
> ```

Yes it works. But as stated above, for those using international keyboards, we lose the capacity to type some useful characters (~, |)...

**Comment by merikan** [comment]
> > For those who use Ghostty, adding the given line to the Ghostty config works
> > ```
> > macos-option-as-alt = true
> > ```
> 
> Yes it works. But as stated above, for those using international keyboards, we lose the capacity to type some useful characters (~, |)...

@sbaro The best way is to keep one `option` key as `option` and then map the other to `alt`. I have kept the right `option` key as `option` and mapped the left `option` key to `alt`.
```
macos-option-as-alt = left
```
